core/components/commerce_cursus/model/commerce_cursus/commerce_cursus.class.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * Commerce Cursus
+ *
+ * Copyright 2023-2024 by modmore <support@modmore.com> & Thomas Jakobi <office@treehillstudio.com>
+ *
+ * @package commerce_cursus
+ * @subpackage classfile
+ */
+
+require_once dirname(__DIR__, 2) . '/vendor/autoload.php';
+
+/**
+ * Class Commerce_Cursus
+ */
+class Commerce_Cursus extends \modmore\Commerce_Cursus\Commerce_Cursus
```

Comment:
What's the purpose of this new empty file that just defines an alias? There's nothing else in `/model/` so I'm confused why we'd need this.

Reply:
The class can be loaded in MODX2 with getService but maybe it is not needed.

Reply:
Removed in https://github.com/modmore/Commerce_Cursus/commit/81bdcd604f1f6ed6944853b4c24ea1ea6b77488b